druid-shell/examples/perftest.rs

```diff
@@ -26,9 +26,12 @@ use time::get_time;
 use kurbo::{Line, Rect};
 use piet::{FillRule, FontBuilder, RenderContext, Text, TextLayoutBuilder};
 
+#[cfg(target_os = "windows")]
+use druid_shell::platform::PresentStrategy;
```

Comment:
Q: What is PresentStrategy?

Reply:
See https://github.com/xi-editor/xi-win/issues/17 . It's part of my quest to make window resizing smooth on Windows. It's not something that's meaningful on other platforms, and most actual users will use the default. It's exposed for experimentation.